### DISCUSSION TITLE
Streaming inference much slower

### DISCUSSION THREAD

**Discussion by canonhui** [fishaudio/fish-speech#616] [Q&A]
i use post_api.py to do inference, and for the exactly same reference audio and to-be-generated-text, when i don't pass the streaming argument, the inference takes like 2 seconds; but if i pass the streaming argument, the inference time increases to like 8 seconds! what could be the reason?

**Comment by AnyaCoder** [comment]
Maybe it triggered `recompilation`. Try to update code to latest commit.
